samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -188,6 +193,7 @@ public void onNewJobModelConfirmed(String version) {
   private void generateNewJobModel() {
     // get the current list of processors
     List<String> currentProcessors = zkUtils.getSortedActiveProcessors();
+    List<String> currentProcessorsPids = zkUtils.getSortedActiveProcessorsPIDs();
```

Comment:
Hmm..ok.. Yeah I think we can handle it in SAMZA-1150. But would it more clear to explicitly handle exception here (say, log it) and then, throw it? I am not sure what is the right approach here.